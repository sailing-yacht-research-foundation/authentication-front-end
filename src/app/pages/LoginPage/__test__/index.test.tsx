import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { LoginPage } from '..';

const shallowRenderer = createRenderer();

describe('<LoginPage />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<LoginPage />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
