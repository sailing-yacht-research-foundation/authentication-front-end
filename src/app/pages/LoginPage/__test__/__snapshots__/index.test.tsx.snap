// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginPage /> should render and match the snapshot 1`] = `
<Row
  align="middle"
  justify="center"
  style={
    Object {
      "minHeight": "100vh",
    }
  }
>
  <Col
    lg={12}
    md={0}
    sm={0}
    xs={0}
  >
    <Curtains$1
      pixelRatio={1}
    >
      <LeftPanel />
    </Curtains$1>
  </Col>
  <Col
    lg={12}
    md={24}
    sm={24}
    xs={24}
  >
    <LoginForm />
  </Col>
</Row>
`;
